### PR TITLE
TASK: Tweak README

### PR DESCRIPTION
**Pull request by kdambekalns**
This converts the `Readme.rst` to Markdown and splits it into two separate files, `README.md` and `CONTRIBUTING.md` as is convention these days.

**Checklist**

- [ ] ~Code follows the PSR-2 coding style~
- [ ] ~Tests have been created, run and adjusted as needed~
- [ ] ~The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)~
- [x] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [x] Reviewer - The first section explains the change briefly for change-logs
- [ ] ~Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions~

### CODE REVIEW

**Comment by kdambekalns** [comment]
TODO:

- [ ] Improve https://github.com/neos/neos-development-distribution?tab=readme-ov-file
- [x] Add some words about `phive` to contribution docs

**Review by ahaeslich — APPROVED** [review]

**Comment by ahaeslich** [comment]
I guess you want to address the checklist in follow up PRs?

**Comment by kdambekalns** [comment]
> I guess you want to address the checklist in follow up PRs?

🤷

What would be the best way to "connect" the READMEs of the dist and collection repositories?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+The normal mode is running PHP locally, but running Mariadb / Postgres in containers (so we know
+we use the right versions etc).
+
+```shell
+bin/behat -c Packages/Neos/Neos.ContentRepository.BehavioralTests/Tests/Behavior/behat.yml.dist
+```
+
+Running all tests can take a long time, depending on the hardware.
+To speed up the process, Behat tests can be executed in a "synchronous" mode by setting the `CATCHUPTRIGGER_ENABLE_SYNCHRONOUS_OPTION` environment variable:
```
Comment (mhsdesign) [inline CONTRIBUTING.md]: The `CATCHUPTRIGGER_ENABLE_SYNCHRONOUS_OPTION` parts in this readme are new and the flag is actually not required anymore as we are moving to sync by default see https://github.com/neos/neos-development-collection/issues/4746

Comment (kdambekalns) [reply CONTRIBUTING.md]: Doh. I _think_ I copied this from the existing README when I split it? 😳

Comment (kdambekalns) [reply CONTRIBUTING.md]: Updated based on the README content I replaced…

**Comment by kdambekalns** [comment]
> I guess you want to address the checklist in follow up PRs?

I guess improving the dev dist readme is a different task…

**Review by mhsdesign — APPROVED** [review]
Thanks, i guess the catchup trigger changes snuck in somehow because your base branch was stale?

**Comment by kdambekalns** [comment]
> Thanks, i guess the catchup trigger changes snuck in somehow because your base branch was stale?

Could be, yes.

**Pull request merged**